### PR TITLE
Changed only description because default of autoupgrade is set to N

### LINKED ISSUE #208: Make no auto upgrade as a default option
The auto upgrade can fail silently, which is very risky; I'd think to turn it off by default in https://github.com/iotexproject/iotex-bootstrap/blob/master/scripts/setup_fullnode.sh

### PR DESCRIPTION
**Pull request by imakowski**
Fixed #208

### CODE REVIEW

**Review by raullenchai — DISMISSED** [review]

File: scripts/setup_fullnode.sh [hunk 1]
```diff
@@ -367,7 +367,7 @@ function main() {
     # Interactive setup phase
     read -p "Do you want to monitor the status of the node [Y/N] (Default: N)? " wantmonitor
     if [ "${_AUTO_UPDATE_}X" != "YX" ];then
-        read -p "Do you want to auto update the node [Y/N] (Default: Y)? " _AUTO_UPDATE_
+        read -p "Do you want to auto update the node [Y/N] (Default: N)? " _AUTO_UPDATE_
```
Comment (huangzhiran) [inline scripts/setup_fullnode.sh:370]: the code at 
scripts/setup_fullnode.sh:466

```
if _AUTO_UPDATE_ == empty string {
   startAutoUpdate 
}
```

maybe scripts/setup_fullnode.sh:466 need changed too

**Review by xianhuawei — DISMISSED** [review]

File: scripts/setup_fullnode.sh (hunk 1 above)
Comment (imakowski) [reply scripts/setup_fullnode.sh:370]: @huangzhiran changed last if. Please review

Comment (imakowski) [reply scripts/setup_fullnode.sh:370]: @xianhuawei Can you review?

**Review by huangzhiran — APPROVED** [review]

**Comment by imakowski** [comment]
@dustinxie @Liuhaai @raullenchai @CoderZhi Can you approve and merge?

**Review by raullenchai — APPROVED** [review]

**Review by xianhuawei — APPROVED** [review]

**Pull request merged**